src/plugins/intel_gna/src/frontend/layer_quantizer.hpp

```diff
@@ -31,6 +31,26 @@ InferenceEngine::Precision GetWeightsPrecision(const LayerInfo& layer_info,
                                                const QuantizedLayerParams& quant_layer_params,
                                                const Config& gna_config);
 
+/**
+ * @brief Returns layer's target biases precision
+ * @param layer_info Layer's information
+ * @param quant_layer_params Layer's quantization parameters
+ * @return layer's target biases precision
+ */
+InferenceEngine::Precision GetBiasesPrecision(const LayerInfo& layer_info,
+                                              const QuantizedLayerParams& quant_layer_params,
+                                              const Config& gna_config);
+
+/**
+ * @brief Checks whether layer's target biases are compound
+ * @param layer_info Layer's information
+ * @param quant_layer_params Layer's quantization parameters
+ * @return true if layer's target biases are compound
+ */
+bool IsBiasCompound(const LayerInfo& layer_info,
```

Comment:
Currently we have four methods outside the class, are they used only in LayerQuantizer?
If they are need for different classes maybe it make sense to put them in separate source/header?

Reply:
Let's leave it as it is for now. They will be probably moved to model quantizer when it's refactored. Scale factor calculator requires refactoring as well.